.github/workflows/package.yml

```diff
@@ -0,0 +1,99 @@
+name: Package Application
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    steps:
+      - name: Retrieves project from the git branch
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat package.json | grep '\"node\":' | sed 's/^ *//;s/ *$//;s/\"node\":\ \"//;s/\",//' )"
+        id: nvm
+
+      - name: Setup node ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+
+      - run: node --version; npm --version; yarn --version
+
+      - name: Cache node modules
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-build-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn install
+
+      - name: Add MacOS certificates
+        if: matrix.os == 'macos-latest'
```

Comment:
Can we skip the MacOS signing when doing pull request builds? There is no reason to sign the builds and it might be faster to build without?

Reply:
There is no flags available to skip signing :(

Reply:
This step should only be necessary for releases right?

Reply:
Well, it's also nice to be able to open any builds vithout having to fight with macOs IMO and if we want to skip signing we would need two configs. There is no config options/flags to avoir this step during build as soon as you provided the config to sign